en/docs/learn/working-with-acr-and-amr.md

```diff
@@ -274,6 +274,49 @@ Follow the steps below to try the scenario without using an application:
 	
 	Note that the AMR values in this example are 	`DemoFaceIdAuthenticator`	, 	`BasicAuthenticator`	, and 	`DemoFingerprintAuthenticator`. These are the authenticators that are used in the authentication process.
 
+#### Translating AMR values
+Follow the steps below to translate the AMR values that get returned to the authentication process.
+
+1. Open the `deployment.toml` file in the `<IS_HOME>/repository/conf directory`.
+
+2. Add the following configurations
+
+    !!! note 
+    You can add any custom AMR mapping to this config.
+
+       ```
+       [[authentication_context.method_refs]]
+       method = "SMSOTP"
+       amr_value = "sms"
+       [[authentication_context.method_refs]]
+       method = "EmailOTP"
+       amr_value = "otp"
+       [[authentication_context.method_refs]]
+       excluded_methods = ["BasicAuthenticator", "AuthenticationMethodToBeIgnored"]
+       ```
+	
+    `method`: Authentication Method Name
+
+    `amr_value`: AMR value mapping you prefer to have as the amr in the ID token. You can refer [AMR draft specification](https://tools.ietf.org/html/draft-ietf-oauth-amr-values-04) for this.
+
+	`excluded_methods`: List of Authentication Methods that you would like to prevent being available on the ID token.
+
+4. Start the identity server
+
+!!! info
```

Comment:
Please double check if this renders properly

Reply:
Fix with a5c790700af9084435520e134ba8f71368868655
<img width="739" alt="Screenshot 2022-04-28 at 9 49 56 PM" src="https://user-images.githubusercontent.com/32576163/165799888-15d6f310-3c1e-4b8d-acce-1ba50491bf89.png">